micronaut-camunda-bpm-feature/src/test/kotlin/info/novatec/micronaut/camunda/bpm/feature/test/DefaultPropertySourceLoaderTest.kt

```diff
@@ -38,17 +39,19 @@ class DefaultPropertySourceLoaderTest {
         @Test
         fun `default value`() {
             assertEquals(50, applicationContext.environment.getProperty("datasources.default.maximum-pool-size", Int::class.java).get())
+            assertEquals(10, applicationContext.environment.getProperty("datasources.default.minimum-idle", Int::class.java).get())
         }
     }
 
     @MicronautTest
     @Nested
+    @Property(name = "datasources.default.maximumPoolSize", value = "42")
```

Comment:
is there any specific reason why you moved the "property" annotation? IMHO both work.

Reply:
Yes, it seems that it works, but if you check the logs you will see that the maximumPoolSize is still 50. Therefore I placed the annotation at the class and now the logs show the correct value too.